acceptance/setup/aio/010_Install.rb

```diff
@@ -31,17 +30,27 @@
       on(master, "rpm -Uvh https://yum.puppetlabs.com/puppet5-release-el-#{version}.noarch.rpm")
     end
   else
-    if ENV['SERVER_VERSION']
-      install_puppetlabs_dev_repo(master, 'puppetserver', ENV['SERVER_VERSION'])
-      install_puppetlabs_dev_repo(master, 'puppet-agent', ENV['SHA'])
+    if ENV['SERVER_VERSION'].nil? || ENV['SERVER_VERSION'] == 'latest'
```

Comment:
How about
```
if ENV['SERVER_VERSION'] && ENV['SERVER_VERSION'] != 'latest' && dev_builds_url == 'http://builds.delivery.puppetlabs.net'
  install_from_build_data_url('puppetserver', "#{dev_builds_url}/puppetserver/#{server_version}/artifacts/#{server_version}.yaml", master)
else
  server_version = ENV['SERVER_VERSION'] || 'latest'

  # builds.delivery.puppetlabs.net can't handle an empty version or a
  # latest version, so we should default to nightlies in those cases
  if dev_builds_url == 'http://builds.delivery.puppetlabs.net'
    dev_builds_url = 'http://nightlies.puppetlabs.com'
  end

  install_puppetlabs_dev_repo(master, project, server_version, nil, :dev_builds_url => dev_builds_url)
  master.install_package('puppetserver')
end

Reply:
I think the current code would still fail if you tried to run with DEV_BUILDS_URL set but SERVER_VERSION not.

Reply:
If SERVER_VERSION isn't set, or is set to 'latest', I'm overwriting dev_builds_url if it's set to builds, so it shouldn't fail. But I'll update this

Reply:
But people who set it to something else, i.e. nightlies.puppetlabs.com, should be able to use latest. I guess just setting latest gets what they want anyway. But mine seems a bit more compact.